gio/ginetaddress.c

```diff
@@ -602,13 +628,19 @@ g_inet_address_new_any (GSocketFamily family)
   g_return_val_if_fail (G_INET_ADDRESS_FAMILY_IS_VALID (family), NULL);
 
   if (family == AF_INET)
-    {    
+    {
       guint8 addr[4] = {0, 0, 0, 0};
 
       return g_inet_address_new_from_bytes (addr, family);
     }
   else
-    return g_inet_address_new_from_bytes (in6addr_any.s6_addr, family);
+    {
+#ifdef HAVE_IPV6
+     return g_inet_address_new_from_bytes (in6addr_any.s6_addr, family);
```

Comment:
Indentation off by one space.

Reply:
Sorry. Having a bit of a nightmare with my editor (vscode), seems it really isn't getting on too well with glib. I have found a fixed a few others too.

https://github.com/frida/glib/pull/6/commits/a31667a6ad4d516a684e16b2375d7aa3fc15fb2d